libutils/string_lib.c

```diff
@@ -1187,6 +1187,19 @@ size_t TrimCSVLineCRLFStrict(char *const data)
     return length;
 }
 
+void StringCloseHole(char *s, int start, int end)
+{
+    assert(s != NULL);
+    assert(0 <= start && start <= end && end <= strlen(s));
+    assert((end - start) <= strlen(s));
+
+    if (end > start)
+    {
+        memmove(s + start, s + end,
+                /* The 1+ ensures we copy the final '\0' */
+                1 + strlen(s + end));
```

Comment:
Just a preference thing: maybe `strlen(...) + 1` instead, since the `NUL`-byte is at the end?

Reply:
Didn't really want to change anything, just move the function. Added assertions, but nothing else.

Reply:
Fixed.